packages/electron-updater/src/differentialDownloader/multipleRangeDownloader.ts

```diff
@@ -97,7 +97,7 @@ function doExecuteTasks(differentialDownloader: DifferentialDownloader, options:
     }
 
     const contentType = safeGetHeader(response, "content-type")
-    const m = /^multipart\/.+?\s*;\s*boundary=(?:"([^"]+)"|([^\s";]+))\s*$/i.exec(contentType)
+    const m = /^multipart\/.+?\s*;[\s]*boundary=(?:"([^"]+)"|([^\s";]+))\s*$/i.exec(contentType)
```

Comment:
Does this need to be adjusted as well?
https://github.com/electron-userland/electron-builder/blob/08773afadfe10911ed1dff084fefe1024c5a74d8/packages/electron-updater/src/differentialDownloader/multipleRangeDownloader.ts#L49